### PR TITLE
ASI-1: Add some new front elements for the PIM.ai connection screen

### PR DESCRIPTION
**Pull request by damien-carcel**
## Description

- Add a new button "slate grey". Corresponding style guide PR: https://github.com/akeneo/pim-docs/pull/955
- Add a new "subtitle" LESS component
- Add a new "Helpcenter-setting" illustration

## Definition Of Done

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/common/title-with-subtitle.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+interface TitleWithSubTitleConfig {
+  main_title: string;
+  sub_title: string;
+  illustration: string;
+  description:  Array<string>;
+}
+
+/**
+ * Setups the connection to PIM.AI.
```
Comment (mmetayer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/common/title-with-subtitle.ts]: ??

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/common/title-with-subtitle.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    this.config = {...this.config, ...options.config};
+  };
+
+  /**
+   * {@inheritdoc}
+   */
+  render(): BaseView {
+    const translatedParagraph: Array<string> = this.config.description.map(translationKey => __(translationKey));
+    const description: string = translatedParagraph.join('</p><p>');
```
Comment (mmetayer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/common/title-with-subtitle.ts]: Couldn't we do that in the template?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/common/title-with-subtitle.ts (hunk 1 above)
Comment (damien-carcel) [reply src/Pim/Bundle/EnrichBundle/Resources/public/js/common/title-with-subtitle.ts]: Heu, oups...

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/common/title-with-subtitle.ts (hunk 2 above)
Comment (damien-carcel) [reply src/Pim/Bundle/EnrichBundle/Resources/public/js/common/title-with-subtitle.ts]: The loop you mean ? I can try.

**Pull request merged**